spec/truffle/launcher_spec.rb

```diff
@@ -85,31 +85,31 @@ def should_print_full_java_command(cmd)
   end
 
   it "takes options from system properties set in JAVA_OPTS" do
-    out = `JAVA_OPTS=-Dpolyglot.ruby.verbosity=2 #{RbConfig.ruby} -e 'puts $VERBOSE'`
+    out = `JAVA_OPTS=-Dpolyglot.ruby.verbosity=true #{RbConfig.ruby} -e 'puts $VERBOSE'`
```

Comment:
We've lost a little bit of functionality then, as before people who were using system properties to set this option could use the same syntax as they could on the command line. But it doesn't matter that much.